frontstage/controllers/party_controller.py

```diff
@@ -659,7 +659,7 @@ def register_pending_shares(payload):
             logger.info("share survey has already been shared, hence ignoring this request.")
         else:
             raise ApiError(logger, response)
-    return response.json()
+    return response
```

Comment:
the :rtype: in the doc string still states dict when it's now a response object

Though i'm not sure you even need to return the response object at all as you don't seem to use it, you could just return a boolean indicating if its successful or not, and maybe an error message if its not.

Mind you what you have will work so I don't mind either way.

Reply:
using response object to check the status code, will modify the doc string